### PR TITLE
Advertize help forum integration blog post

### PR DESCRIPTION
**Pull request by wm75**

### CODE REVIEW

**Comment by arash77** [comment]
We must ignore importing the plugins in the preview mode if we don't need them.

**Comment by wm75** [comment]
@arash77 seems I forgot to import re in #28, sorry about that.
Running out of battery here, can you fix it?

**Comment by arash77** [comment]
Or maybe we get this because of [spamming](https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#create-an-issue-comment--status-codes).

**Comment by wm75** [comment]
Quite possible since we're trying to create comments with the same content repeatedly.

**Comment by arash77** [comment]
Fixed in #35

**Comment by bgruening** [comment]
rebased, lets see.

File: posts/2024/2024-06-10_help_forum_integration.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+media:
+  - mastodon-eu-freiburg
+
+hashtags:
+  mastodon:
```
Comment (arash77) [inline posts/2024/2024-06-10_help_forum_integration.md:6]: This line should also be the same as media name `mastodon-eu-freiburg`

**Review by arash77 — CHANGES_REQUESTED** [review]

File: posts/2024/2024-06-10_help_forum_integration.md (hunk 1 above)
Comment (bgruening) [reply posts/2024/2024-06-10_help_forum_integration.md:6]: please add a suggestion @arash77 https://haacked.com/archive/2019/06/03/suggested-changes/

This way its super easy to accept it and you get credits for it.

Comment (arash77) [inline posts/2024/2024-06-10_help_forum_integration.md:6]:
```suggestion
  mastodon-eu-freiburg:
```

**Review by arash77 — CHANGES_REQUESTED** [review]

**Comment by bgruening** [comment]
I will leave this to Wolfgang, my question is why the bot actually posted the preview and did not fail?

Would merging this as is, post anything?

**Comment by arash77** [comment]
>why the bot actually posted the preview and did not fail?

I will add a code to fail when the mentions and hashtags for the media do not exist.

> Would merging this as is, post anything?

Yes, but it just ignores the mentions and hashtags.

**Comment by bgruening** [comment]
I see. Yes we should be aware of all kind of user errors :)

**Comment by arash77** [comment]
> I see. Yes we should be aware of all kind of user errors :)

Should we create a comment in PR for this kind of error or an error raise would be enough?
- Metadata schema is not right.
- Missing/Invalid media in metadata.
- Invalid mentions/hashtags media in metadata

**Comment by arash77** [comment]
Showing this kind of error in #45

**Comment by arash77** [comment]
It seems to be a bug that old posts are also show in the preview!

**Comment by wm75** [comment]
Cool, now let me try to fix and update this.

**Comment by wm75** [comment]
Finally!
@bgruening you can now review the actual content of this if you want to :-)

**Review by arash77 — APPROVED** [review]

**Review by bgruening — APPROVED** [review]
I like the content. Only remark is if it would be nice to create a new PR so we use the clean PR history to link in documentarions etc as a good example.

**Comment by wm75** [comment]
Closing in favor of #50

**Pull request closed without merging**